packages/admin-next/dashboard/src/hooks/api/collections.tsx

```diff
@@ -90,19 +90,24 @@ export const useUpdateCollection = (
 export const useUpdateCollectionProducts = (
   id: string,
   options?: UseMutationOptions<
-    ProductCollectionRes,
-    Error,
-    UpdateProductCollectionProductsReq
+    { collection: HttpTypes.AdminCollection },
```

Comment:
note: I ve seen that we have some types that include `Response` at the end of the types and other are not, we will have to align. not for this pr but more of a general comment :D

Reply:
If we exported the the type as `{ collection: AdminCollection }` then I think it should be suffixed with Response. But none of the other recent ones are exported like that, only the `Admin<Entity>` is exported, so I stuck with that for this PR as well. But yea we need to align on this, for sure.